test/emqx_dashboard_cli_SUITE.erl

```diff
@@ -4,11 +4,70 @@
 
 -compile(export_all).
 
+all() ->
+    [cli].
 
-all() -> 
-    [].
+init_per_suite(Config) ->
+    [start_apps(App, {SchemaFile, ConfigFile}) ||
+        {App, SchemaFile, ConfigFile}
+            <- [{emqx, local_path("deps/emqx/priv/emqx.schema"),
+                       local_path("deps/emqx/etc/emqx.conf")},
+                {emqx_management, local_path("deps/emqx_management/priv/emqx_management.schema"),
+                                  local_path("deps/emqx_management/etc/emqx_management.conf")},
+                {emqx_dashboard, local_path("priv/emqx_dashboard.schema"),
+                                 local_path("etc/emqx_dashboard.conf")}]],
```

Comment:
Duplicate with the procedure in another SUITE. I suggest you to abstract the common logic.